<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>723</width>
    <height>511</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Search Imagery</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>331</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Select the imagery you want to download</string>
   </property>
  </widget>
  <widget class="QLabel" name="catalog_url_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>601</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Catalog URL: </string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>70</y>
     <width>321</width>
     <height>371</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayoutQueries">
    <property name="spacing">
     <number>3</number>
    </property>
    <property name="margin">
     <number>3</number>
    </property>
    <item>
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayoutQueries">
      <item row="0" column="0">
       <widget class="QCheckBox" name="checkBoxLocation">
        <property name="text">
         <string>Location</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDateEdit" name="dateEditAcquisitionFrom"/>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditLocation"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="labelNumImages">
        <property name="text">
         <string>Numbers of images</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="checkBoxAcquisitionFrom">
        <property name="text">
         <string>Acquisition Date From</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEditResolutionFrom"/>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="labelOrderBy">
        <property name="text">
         <string>Order By</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="checkBoxResolutionFrom">
        <property name="text">
         <string>Resolution From</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QComboBox" name="comboBoxOrderBy">
        <item>
         <property name="text">
          <string>Acquisition Date</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>GSD</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="checkBoxAcquisitionTo">
        <property name="text">
         <string>Acquisition Date To</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDateEdit" name="dateEditAcquisitionTo"/>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="lineEditResolutionTo"/>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="lineEditNumImages"/>
      </item>
      <item row="4" column="0">
       <widget class="QCheckBox" name="checkBoxResolutionTo">
        <property name="text">
         <string>Resoultion To</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QRadioButton" name="radioButtonAsc">
        <property name="text">
         <string>Ascending</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QRadioButton" name="radioButtonDesc">
        <property name="text">
         <string>Descending</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayoutSearch">
      <item>
       <widget class="QPushButton" name="pushButtonSearch">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>35</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Search</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>351</width>
     <height>371</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayoutListWidget">
    <property name="margin">
     <number>3</number>
    </property>
    <item>
     <widget class="QListWidget" name="listWidget"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pbSetDefault">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>460</y>
     <width>161</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Set Queries As Default</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbLoadDefault">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>460</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Load Default Queries</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
